samples/hello-resub/src/tutorials.md

```diff
@@ -0,0 +1,77 @@
+
+## install resub
+    D:\lib\reactxp\samples\hello-resub>yarn add resub@latest
+
+## Add Store named HelloStore
+  add a new file in src fold,and named it with helloStore.ts
+    import { StoreBase, AutoSubscribeStore, autoSubscribe } from 'resub';
+
+    @AutoSubscribeStore
+    //store in resub, is a class extends from StoreBase.
+    class HelloStore extends StoreBase {
+
+        //_helloString:string ,should give a type here
+        private _helloString: string = 'hello resub!';
+
+        @autoSubscribe
+        // this method should return a string type
+        getHello():string {
+            return this._helloString;
+        }
+    }
+
+    // export a instance of class HelloStore
+    export = new HelloStore();
+
+
+
+## Add Component With HelloStore
+    add a new file in src fold,and named it with hello.tsx
+    import { ComponentBase } from 'resub';
+    import RX = require('reactxp');
+
+    // should use require? for export=
+    import HelloStore =require('./helloStore');
+
+    interface HelloState {
+        Hello?: string;
+    }
+
+    class Hello extends ComponentBase<{}, HelloState> {
+        protected _buildState(props: {}, initialBuild: boolean): HelloState {
+            return {
+                Hello: HelloStore.getHello()
+            }
+        }
+
+        render(): JSX.Element | null  {
+            return (
+                <RX.Text>
+                    { this.state.Hello }
+                </RX.Text>
+            );
+        }
+    }
+
+    // notice it
+    export = Hello;
+
+## show hello component in app.tsx
+  import Hello =require('./hello');
+  and then add hello component in render function
+                <RX.Text style={ styles.welcome }>
+                    Welcome to ReactXP
+                </RX.Text>
+                <Hello >                    
+                </Hello>
+
+
+
+## build it
+    D:\lib\reactxp\samples\hello-resub>yarn run web-watch
+    a compile error here:
+    ERROR in [at-loader] ./node_modules/resub/dist/src/StoreBase.d.ts:13:20

```

Comment:
What about suggesting a viable workaround for the ones that just stepped into reactxp/typescript? (like me)

After taking a look at [tsconfig.json compiler options](https://www.typescriptlang.org/docs/handbook/compiler-options.html) I got rid of it by setting this options in tsconfig.json
```diff
+    "noImplicitAny": false,
+    "skipLibCheck": true,
```
> I've not looked into which one of the above two changes is the error not being checked or ignored.

**BUT** I'd like to have those checks being applied in my own code sooner or later. Is there a better workaround / correct solution?